### PR TITLE
Week 2 | Request Tris

### PR DESCRIPTION
**Pull request by dacayant**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 1]
```diff
@@ -10,9 +10,14 @@
...
 public class HelloWorldServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
+
+    @Override
```
Comment (imichaelnorris) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:18]: This should be at the same indentation level of public void.

File: walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java [hunk 2]
```diff
@@ -24,7 +24,11 @@
 @WebServlet("/page-views")
 public class PageViewServlet extends HttpServlet {
 
-  private int pageViews = 0;
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
```
Comment (imichaelnorris) [inline walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java]: Where is serialVersionUID used?

**Review by imichaelnorris — CHANGES_REQUESTED** [review]
Usually files with the name "test" in them are unit tests.

fetchTest.js contains the displayHello function which is calling your endpoint and then updating the greeting container. Let's rename the file to something like "hello.js."

File: walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java (hunk 2 above)
Comment (dacayant) [reply walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java]: When I used to run the mvn command in the console, I would get a serialization error. In the .java file, there was also a yellow underline under the class name of the program and it said "The serializable class HelloWorldServlet does not declare a static final serialVersionUID field of type long." In the end, I clicked the quick fix option and the mvn command ran without errors.

**Comment by dacayant** [comment]
I made the changes based on your comments. Are they all good now?

I also made the same changes in the other branches after the creation of this pull request.

**Comment by imichaelnorris** [comment]
Yep, looks good to me.

**Pull request merged**